frontend/app/routes/_gcweb-app.personal-information.home-address.suggested.tsx

```diff
@@ -0,0 +1,154 @@
+   */
+  const suggestedAddressInfo = await getAddressService().getAddressInfo(userId, userInfo?.mailingAddress ?? '');
+  const formDataRadio = Object.fromEntries(await request.formData());
+  //retrieve selected address, store it in the session and then redirect to the confirm page...
+  console.debug('SELECTED :: ' + formDataRadio.selectedAddress);
+  const session = await sessionService.getSession(request.headers.get('Cookie'));
+  if (formDataRadio.selectedAddress == 'home') {
+    session.set('newHomeAddress', homeAddressInfo);
+  } else {
+    session.set('newHomeAddress', suggestedAddressInfo);
+  }
+  return redirect('/personal-information/home-address/confirm', {
+    headers: {
+      'Set-Cookie': await sessionService.commitSession(session),
+    },
+  });
+}
+
+/**
+ * MAIN
+ * @returns
+ */
+export default function HomeAddressSuggested() {
+  //const actionData = useActionData<typeof action>();
+
+  const { homeAddressInfo, suggestedAddressInfo } = useLoaderData<typeof loader>();
+  const { t } = useTranslation(i18nNamespaces);
+
+  return (
+    <>
+      <Form method="post">
+        <p>{t('personal-information:home-address.suggested.note')}</p>
+        <div className="grid gap-6 md:grid-cols-2">
+          <PersonalInformationSection title={t('personal-information:home-address.suggested.address-entered')} icon="glyphicon-map-marker">
+            {homeAddressInfo && <Address address={homeAddressInfo?.address} city={homeAddressInfo?.city} provinceState={homeAddressInfo?.province} postalZipCode={homeAddressInfo?.postalCode} country={homeAddressInfo?.country} />}
+            {!homeAddressInfo && <p>{t('personal-information:index.no-address-on-file')}</p>}
+          </PersonalInformationSection>
+          <PersonalInformationSection title={t('personal-information:home-address.suggested.address-suggested')} icon="glyphicon-map-marker">
+            {suggestedAddressInfo ? (
+              <Address address={suggestedAddressInfo?.address} city={suggestedAddressInfo?.city} provinceState={suggestedAddressInfo?.province} postalZipCode={suggestedAddressInfo?.postalCode} country={suggestedAddressInfo?.country} />
+            ) : (
+              <p>{t('personal-information:index.no-address-on-file')}</p>
+            )}
+          </PersonalInformationSection>
+        </div>
+        <div>
+          <InputRadios
+            id="selected-address"
+            name="selectedAddress"
+            legend={t('personal-information:home-address.suggested.choose-address')}
+            options={[
+              { value: 'home', children: t('personal-information:home-address.suggested.use-entered') },
+              { value: 'suggested', children: t('personal-information:home-address.suggested.use-suggested') },
+            ]}
+            required
+          />
+        </div>
+        <p>{t('personal-information:home-address.suggested.re-enter-address')}</p>
```

Comment:
I feel there should be a `<Link>` somewhere in this text that takes the user to the edit address form page, possibly around "re-enter your address".

Reply:
a new link has been added to send the user back to the edit page